xAPI-Communication.md

```diff
@@ -1667,246 +1667,18 @@ described in [Appendix A: Converting Statements to 1.0.0](#Appendix3A).
 
 ###### <a name="4.0.s1"></a>Rationale
 
-In order to balance interoperability and the varying security requirements of different environments, several 
-authentication options are defined.
+In order to support the varying security requirements of different environments, a specific authentication mechanisim is not defined. In practice [HTTP Basic Authentication](http://tools.ietf.org/html/rfc7235) has been very broadly used, so it may be worth considering as a lowest common denominator.
```

Comment:
Calling out Basic Auth may be controversial, but one criticism I've heard of of the "remove the auth language" idea is that it will decrease interoperability, and I think in practice the inertia towards basic auth counters that concern well.

Reply:
In general, I agree with this approach. We should bring it to the broader WG and see if there is any push back.

Reply:
I like this approach but think it could be stronger. Specifically, instead of "so it may be worth considering as a lowest common denominator", I think we should be clear that LRSs need to support Basic HTTP in order to pass conformance testing, because that's what the conformance test uses.

Reply:
+1 @garemoko -- That provides a clear reason.

Reply:
```suggestion
In order to support the varying security requirements of different environments, a specific authentication mechanism is not defined. In practice [HTTP Basic Authentication](http://tools.ietf.org/html/rfc7235) has been very broadly used, so it may be worth considering as a lowest common denominator.
```
Typo.

If it is required to pass conformance testing, calling it out seems appropriate, but it may be prudent to add other "obviously compatible" security standards as possibilities that are not covered by the standard: OAuth2, Client side certificates, etc.